### ISSUE TITLE
[BUG] Daedalus 5.3.0 gets stuck with the infamous ledger bug of cardano-node v1.35.4

### ISSUE THREAD

**Issue by onyxstakepool** [input-output-hk/daedalus#3182]
**Daedalus 5.3.0** gets stuck in epoch 484 around slot 391,468 and cardano-node v1.35.4 keeps crashing with the [infamous ledger bug](https://forum.cardano.org/t/network-outage-news/113418) that was fixed in [cardano-node v1.35.5](https://github.com/IntersectMBO/cardano-node/releases/tag/1.35.5) on Jan 27, 2023!

```
cardano-node: Failure in Data.Map.balanceR
CallStack (from HasCallStack):
  error, called at src/Data/Map/Internal.hs:4157:30 in containers-0.6.5.1-EiES0HFUZ8PBGNrpVjoYRF:Data.Map.Internal
```

**Emergency Fix on MacOS**
Daedalus 5.3.0 can be fixed by copying the **cardano-cli** and **cardano-node** binaries from the [cardano-node v1.35.5 binary distribution](https://update-cardano-mainnet.iohk.io/cardano-node-releases/cardano-node-1.35.5-macos.tar.gz) to the directory **Contents/MacOS/** in the **Daedalus Mainnet.app** application.

**Comment by kevinhammond** [comment]
Thank you, @onyxstakepool.  The Daedalus team is looking into this.  I've pointed them at this issue.

**Comment by glitch40** [comment]
Emergency Fix for Windows

Daedalus 5.3.0 can be fixed by extracting and copying the cardano-cli.exe and cardano-node.exe files from the [cardano-node v1.35.5 distribution](https://update-cardano-mainnet.iohk.io/cardano-node-releases/cardano-node-1.35.5-win64.zip) to the directory C:\Program Files\Daedalus Mainnet  with Daedalus closed. You will need to give both files Admin privileges then restart and allow Daedalus to sync. 

(1.35.7 cardano-node and cardano-cli will also work from my testing)

**Comment by michalrus** [comment]
This has been fixed in the latest 5.3.1 release. Feel free to reopen if you're still experiencing issues.
